src/pages/[platform]/deploy-and-host/sandbox-environments/features/index.mdx

```diff
@@ -168,17 +168,88 @@ npx amplify sandbox --identifier feature1sandbox secret set baz
+npx amplify sandbox --outputs-out-dir path/to/config --outputs-format ["json", "dart"]
 ```
 
 Alternatively, if you want to generate the config for a branch environment to test against, run the following command in the terminal.
 
 ```bash
 npx amplify generate config --app-id <AMPLIFY_APP_ID> --branch main --format ["mjs", "json", "json-mobile", "ts", "dart"] out-dir ./path/to/config
 ```
+</InlineFilter>
+
+<InlineFilter filters={['flutter']}>
+```bash title="Terminal" showLineNumbers={false}
+npx amplify sandbox --outputs-format dart --outputs-out-dir lib
+```
+</InlineFilter>
+<InlineFilter filters={['android']}>
+<Callout warning>
+  Be sure to add a "raw" folder under app/src/main/res directory if it doesn't
+  exist.
+</Callout>
+
+```bash title="Terminal" showLineNumbers={false}
+npx amplify sandbox --outputs-format json-mobile --outputs-out-dir app/src/main/res/raw
+```
+</InlineFilter>
+
+<InlineFilter filters={['swift']}>
+
+```bash title="Terminal" showLineNumbers={false}
+npx amplify sandbox --outputs-format json-mobile
+```
+
+Once the sandbox environment is running, you would also generate the configuration files for your application. However, Xcode won't be able to recognize them. For recognizing the files, you need to drag and drop the generated files to your project.
+
+<video autoPlay={true} muted={true} loop={true} width="100%" playsInline={true}>
+  <source src="/images/gen2/getting-started/ios/ios-getting-started-2.mp4" />
+</video>
+</InlineFilter>
+
+### Deployment Environment
+
+Alternatively, if you want to generate the config for a branch environment to test against, you can run the following command below in the terminal:
+
+<InlineFilter filters={['android','angular','flutter','javascript','nextjs','react','react-native','swift','vue']}>
+For Web and React Native, generating the config with the default format and output directory.
+
+```bash title="Terminal" showLineNumbers={false}
+npx amplify generate config --app-id <app-id> --branch main
+```
+</InlineFilter>
+<InlineFilter filters={['flutter']}>
+```bash title="Terminal" showLineNumbers={false}
+npx amplify generate config --app-id <app-id> --branch main --format dart --out-dir lib
+```
+</InlineFilter>
+<InlineFilter filters={['android']}>
+<Callout warning>
+  Be sure to add a "raw" folder under app/src/main/res directory if it doesn't
+  exist.
+</Callout>
+
+```bash title="Terminal" showLineNumbers={false}
+npx amplify generate config --app-id <app-id> --branch main --format json-mobile --out-dir app/src/main/res/raw
+```
+</InlineFilter>
+<InlineFilter filters={['swift']}>
+
+```bash title="Terminal" showLineNumbers={false}
+npx amplify generate config --app-id <app-id> --format json-mobile
+```
+
+Once the sandbox environment is running, you would also generate the configuration files for your application. However, Xcode won't be able to recognize them. For recognizing the files, you need to drag and drop the generated files to your project.
```

Comment:
```suggestion
Once the sandbox environment is running, it will generate the backend outputs file for your frontend application. However, Xcode won't be able to recognize them. For recognizing the files, you need to drag and drop the generated files to your project.
```